pkgs/top-level/linux-kernels.nix

```diff
@@ -884,5 +892,28 @@ in
       '';
     };
 
+  # Merge two kernel configs
+  mergeConfigs =
+    {
+      src,
+      name ? "merged.config",
+      version ? (builtins.parseDrvName src.name).version,
+      a,
+      b,
```

Comment:
IMO it would either make more sense to take two positional arguments, or ideally a list that could contain more than two.